SDWebImage/SDWebImageCodersManager.m

```diff
@@ -65,12 +67,6 @@ - (void)removeCoder:(nonnull id<SDWebImageCoder>)coder {
 - (NSArray<SDWebImageCoder> *)coders {
     __block NSArray<SDWebImageCoder> *sortedCoders = nil;
     dispatch_sync(self.mutableCodersAccessQueue, ^{
-        if (self.mutableCoders.count == 0) {
```

Comment:
Well...No more nil protect. It's that OK ? 😄

Reply:
As I said in the description, I think it is bad to do non-standard things inside a getter. If the user decides to empty the array for whatever reason, I don't see any issues. On the other hand, this auto-reset can generate unexpected behaviour.